cypress/e2e/room/room-header.spec.ts

```diff
@@ -0,0 +1,182 @@
+        cy.createRoom({ name: "Test Room" }).viewRoomByName("Test Room");
+
+        cy.findByRole("button", { name: "Room info" })
+            .click() // Highlight the button
+            .then(($btn) => {
+                // Note it is not possible to get CSS values of a pseudo class with "have.css".
+                const color = $btn[0].ownerDocument.defaultView // get window reference from element
+                    .getComputedStyle($btn[0], "before") // get the pseudo selector
+                    .getPropertyValue("background-color"); // get "background-color" value
+
+                // Assert the value is equal to $accent == hex #0dbd8b == rgba(13, 189, 139)
+                expect(color).to.eq("rgb(13, 189, 139)");
+            });
+
+        cy.get(".mx_RoomHeader").percySnapshotElement("Room header - with a highlighted button", {
+            widths: [300, 600], // Magic numbers
+        });
+    });
+
+    describe("with a video room", () => {
+        const createVideoRoom = () => {
+            // Enable video rooms. This command reloads the app
+            cy.setSettingValue("feature_video_rooms", null, SettingLevel.DEVICE, true);
+
+            cy.get(".mx_LeftPanel_roomListContainer", { timeout: 20000 })
+                .findByRole("button", { name: "Add room" })
+                .click();
+
+            cy.findByRole("menuitem", { name: "New video room" }).click();
+
+            cy.findByRole("textbox", { name: "Name" }).type("Test video room");
+
+            cy.findByRole("button", { name: "Create video room" }).click();
+
+            cy.viewRoomByName("Test video room");
+        };
+
+        it("should render buttons for room options, beta pill, invite, chat, and room info", () => {
+            createVideoRoom();
+
+            cy.get(".mx_RoomHeader").within(() => {
+                cy.findByRole("button", { name: "Room options" }).should("be.visible");
+                cy.findByRole("button", { name: "Video rooms are a beta feature Click for more info" }).should(
+                    "be.visible",
+                ); // Beta pill
+                cy.findByRole("button", { name: "Invite" }).should("be.visible");
+                cy.findByRole("button", { name: "Chat" }).should("be.visible");
+                cy.findByRole("button", { name: "Room info" }).should("be.visible");
+
+                // Assert that there is not a button except those buttons
+                cy.findAllByRole("button").should("have.length", 5);
+            });
+
+            cy.get(".mx_RoomHeader").percySnapshotElement("Room header - with a video room", {
+                widths: [300, 600], // Magic numbers
+            });
+        });
+
+        it("should render a working chat button which opens the timeline on a right panel", () => {
+            createVideoRoom();
+
+            cy.get(".mx_RoomHeader").findByRole("button", { name: "Chat" }).click();
+
+            // Assert that the video is rnedered
```

Comment:
Tiny typo here in `rnedered`

Reply:
Thanks! Fixed with https://github.com/matrix-org/matrix-react-sdk/pull/10713/commits/d7a700dca1e32c50400eb01318f91e62889062a1